pkg/config/config.go

```diff
@@ -259,8 +260,8 @@ func GetFlatIPAM(isControlLoop bool, IPAM *types.IPAMConfig, extraConfigPaths ..
 			return flatipam, foundflatfile, err
 		}
 	}
-	var err error
-	return flatipam, foundflatfile, err
+
+	return flatipam, foundflatfile, errors.New("no config file was found")
```

Comment:
thanks , good catch.

Would you mind also changing line [260](https://github.com/k8snetworkplumbingwg/whereabouts/pull/323/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R260) to `return flatipam, foundflatfile, nil` ? I think that's more explicit.